src/sidebar/components/thread-list.js

```diff
@@ -46,16 +51,20 @@ function getScrollContainer() {
 function ThreadList({ thread }) {
   // Height of the visible area of the scroll container.
   const [scrollContainerHeight, setScrollContainerHeight] = useState(
-    getScrollContainer().clientHeight
+    window.innerHeight
   );
 
   // Scroll offset of scroll container. This is updated after the scroll
   // container is scrolled, with debouncing to limit update frequency.
   // These values are in multiples of `SCROLL_PRECISION` to optimize
   // for performance.
-  const [scrollPosition, setScrollPosition] = useState(
-    getScrollContainer().scrollTop
-  );
+  const [scrollPosition, setScrollPosition] = useState(0);
+
+  useLayoutEffect(() => {
+    const scrollContainer = getScrollContainer();
+    setScrollContainerHeight(scrollContainer.clientHeight);
+    setScrollPosition(roundScrollPosition(scrollContainer.scrollTop));
+  }, []);
```

Comment:
This layout effect sets the initial values of `scrollContainerHeight` and `scrollPosition` after the first render.